src/app/components/PortraitVideoModal/index.tsx

```diff
@@ -104,25 +104,31 @@ const PortraitVideoModal = ({
     if (previous) {
       const [fallbackImage, portraitImage] = previous?.images || [];
 
-      player.setPreviousPlaylist({
-        title: previous?.video?.title ?? '',
-        holdingImageURL: setImageWidth(
-          (portraitImage || fallbackImage)?.urlTemplate,
-        ),
-        items: [{ versionID: previous?.video?.version?.id }],
-      });
+      player.setPreviousPlaylist(
+        {
+          title: previous?.video?.title ?? '',
+          holdingImageURL: setImageWidth(
+            (portraitImage || fallbackImage)?.urlTemplate,
+          ),
+          items: [{ versionID: previous?.video?.version?.id }],
+        },
+        { statsObject: { clipPID: previous?.video?.id } },
```

Comment:
Nice, wondering though, if this is too lossy for the `statsObject` and if it would be worth passing it like this for completeness to prevent breakages in data

```
    statsObject: {
      ...player?.statsObject,
      clipPID: next?.video?.id,
    },
    
    ```